components/swim/Cargo.lock

```diff
@@ -0,0 +1,313 @@
+[root]
```

Comment:
This lockfile shouldn't be in version control since we're using a common workspace. I see that you've got the right line in your `Cargo.toml` to enable the workspace so this was probably generated before you added that and can be safely deleted